lifetimes/utils.py

```diff
@@ -433,53 +440,74 @@ def expected_cumulative_transactions(model, transactions, datetime_col, customer
         the number of time units since the begining of
         data for which we want to calculate cumulative transactions
     datetime_format: string, optional
-        a string that represents the timestamp format. Useful if Pandas can't understand
-        the provided format.
+        a string that represents the timestamp format. Useful if Pandas can't
+        understand the provided format.
     freq: string, optional
         Default 'D' for days, 'W' for weeks, 'M' for months... etc. Full list here:
         http://pandas.pydata.org/pandas-docs/stable/timeseries.html#dateoffset-objects
     set_index_date: bool, optional
         when True set date as Pandas DataFrame index, default False - number of time units
+    div_coef: float, optional
```

Comment:
this is a better docstring than the one above - want to copy this up there too?

Reply:
if `div_coef` is a float, an expression like `i % div_coef == 0` is ambiguous about it's output, no?

Reply:
Good catch it should be int with current logic.